Automattic/wp-calypso
# Pricing Page Rework: Add Jetpack Social product

#### Proposed Changes

This PR adds the necessary data to display the "Social" product in the pricing grid.

Implementation notes:
- This PR also fixes an issue in #67154 where the Boost product would be displayed twice (for a logged in user) if the site already owned Boost.

#### Testing Instructions

* click on Jetpack Cloud live link below
    * Goto `/pricing?flags=jetpack/pricing-page-rework-v1`
* or boot up this PR 
    * Run `git fetch && git checkout add/social-product-data`
    * Run `yarn start-jetpack-cloud`
    * Goto http://jetpack.cloud.localhost:3000/pricing?flags=jetpack/pricing-page-rework-v1
* View the products in the pricing page and verify that the "Social" product is now displaying.

Note: I believe the price for the Social product is incorrect. The price will need to be modified in a separate patch, or possibly by Marketing, or the Reach team.

#### Screenshot

![Markup 2022-08-30 at 19 23 48](https://user-images.githubusercontent.com/11078128/187560706-313f2a59-7a91-4551-b9d0-69706c1e5269.png)


#### Pre-merge Checklist






Completes: 1202796695664022-as-1202886227656202/f

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1202886227656202
